scripts/test_translation_templates.py

```diff
@@ -0,0 +1,127 @@
+import coloredlogs
+
+import os
+import docker
+import sys
+import subprocess
+import tempfile
+import logging
+
+logger = logging.getLogger(__name__)
+
+coloredlogs.install(level='DEBUG', logger=logger, fmt='%(message)s')
+
+SCHEMAS_TO_EXTRACT = ["census_individual_gb_wls", "census_individual_gb_nir"]
+
+
+def get_template_content(filename, ignore_context=False):
+    line_beginnings_to_ignore = ['"POT-Creation-Date']
+
+    if ignore_context:
+        line_beginnings_to_ignore += ["#:"]
+
+    with open(filename) as file:
+        return "".join(
+            filter(
+                lambda l: all(
+                    not l.startswith(param) for param in line_beginnings_to_ignore
+                ),
+                file.readlines(),
+            )
+        )
+
+
+def print_filename_results(filename, success=True):
+    if success:
+        logger.debug(f"{filename} - NO CHANGES")
+    else:
+        logging.error(f"{filename} - CHANGES FOUND")
+
+
+def build_static_template(output_filepath):
+    subprocess.run(
+        [
+            'pipenv',
+            'run',
+            'pybabel',
+            'extract',
+            '-F',
+            'babel.cfg',
+            '-o',
+            output_filepath,
+            '.',
+        ]
+    )
+
+
+def check_static_template(source_dir, target_dir):
+    source_contents = get_template_content(
+        f"{source_dir}/messages.pot", ignore_context=True
+    )
+    target_contents = get_template_content(
+        f"{target_dir}/messages.pot", ignore_context=True
+    )
+
+    success = source_contents == target_contents
+
+    print_filename_results(f"{source_dir}/messages.pot", success)
+
+    return 0 if success else 1
+
+
+def build_schema_templates(output_dir):
+    docker_client = docker.from_env()
+
+    for schema_name in SCHEMAS_TO_EXTRACT:
+        template_file = f"{schema_name}.pot"
+        schema_file = f"{schema_name}.json"
+
+        docker_client.images.pull('onsdigital/eq-translations:latest')
+        run_results = docker_client.containers.run(
+            'onsdigital/eq-translations',
+            f"pipenv run python -m cli.template_extractor ../eq-survey-runner/data/en/{schema_file} /usr/src/templates",
+            volumes={
+                os.getcwd(): {'bind': '/usr/src/eq-survey-runner', 'mode': 'rw'},
+                output_dir: {'bind': '/usr/src/templates', 'mode': 'rw'},
+            },
+            remove=True,
+        )
+        logger.info(f"Building {output_dir}/{template_file}")
+        logger.info(run_results.decode())
+
+
+def check_schema_templates(source_dir, target_dir):
+    failure_count = 0
```

Comment:
Since this is not outputted anywhere, can we just use a boolean?
```python
def check_schema_templates(source_dir, target_dir):
    is_valid = True

    for schema_name in SCHEMAS_TO_EXTRACT:
        template_file = f"{schema_name}.pot"

        source_contents = get_template_content(f"{source_dir}/{template_file}")
        target_contents = get_template_content(f"{target_dir}/{template_file}")
        is_valid = source_contents == target_contents

        print_filename_results(f"app/translations/{template_file}", is_valid)

    return is_valid
```